docs/usage/ruby.md

```diff
@@ -1,29 +1,23 @@
 ---
-title: Ruby Bundler Support
+title: Ruby Bundler support
 description: Ruby Bundler support in Renovate
 ---
 
-# Automated Dependency Updates for Ruby Bundler Dependencies
+# Automated dependency updates for Ruby Bundler dependencies
 
 Renovate supports upgrading dependencies in Bundler's `Gemfile`s and their accompanying `Gemfile.lock` files.
 Support is considered "alpha" stage until there have been some more real-world tests.
 
-## How It Works
+## How it works
 
-1.  Renovate will search each repository for any `Gemfile` files
-2.  Existing dependencies will be extracted from the files
-3.  Renovate will resolve the dependency on Rubygems or elsewhere if configured, and look for any newer versions
-4.  A PR will be created with `Gemfile` and `Gemfile.lock` updated in the same commit
-5.  If the source repository has either a "changelog" file or uses GitHub releases, then Release Notes for each version will be embedded in the generated PR
+1. Renovate searches in each repository for any `Gemfile` files
+1. Existing dependencies are extracted from the `Gemfile`'s
```

Comment:
Plural, not possessive?

```suggestion
1. Existing dependencies are extracted from the `Gemfile`s
```

Reply:
Also, is it "Gemfile files" or "Gemfiles"? Honestly, I'm not sure.

Reply:
Let's use Gemfiles, that's what https://bundler.io/ uses on their own webpage. 😄